src/Compilers/Test/Utilities/VisualBasic/ParserTestUtilities.vb

```diff
@@ -10,8 +10,10 @@ Imports Microsoft.CodeAnalysis.Text
 Imports Microsoft.CodeAnalysis.VisualBasic.SyntaxFacts
 Imports Roslyn.Test.Utilities
 Imports Xunit
+Imports Microsoft.CodeAnalysis.Collections
 
 Friend Module ParserTestUtilities
+    Friend ReadOnly Property PooledStringBuilderPool As ObjectPool(Of PooledStringBuilder) = PooledStringBuilder.CreatePool(If(Environment.Is64BitProcess, 96, 64))
```

Comment:
>If(Environment.Is64BitProcess, 96, 64) [](start = 124, length = 38)

It is not clear why would the size of the pool depend on bit-ness of the process. It is not measured in bytes, but in number of instances that a kept for reuse.